src/test/java/teammates/test/cases/browsertests/AdminHomePageUiTest.java

```diff
@@ -167,10 +167,17 @@ private void testCreateInstructorAction() throws Exception {
         confirmationPage = AppPage.createCorrectLoginPageType(browser)
                            .loginAsJoiningInstructor(TestProperties.TEST_INSTRUCTOR_ACCOUNT,
                                                      TestProperties.TEST_INSTRUCTOR_PASSWORD);
-        confirmationPage.clickConfirmButton();
+        try {
+            confirmationPage.clickConfirmButton();
+        } catch (Exception e) {
+            if (!confirmationPage.getPageSource().contains("TEAMMATES could not locate what you were trying to access.")) {
```

Comment:
Is this string literal defined somewhere in the code base? It's better to refer to that to reduce the brittleness of this code.